blog/electron-21-0.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Electron 21.0.0
+date: 2022-09-27T00:00:00.000Z
+authors:
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+    - name: vertedinde
+      url: 'https://github.com/vertedinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+    - name: georgexu99
+      url: 'https://github.com/georgexu99'
+      image_url: 'https://github.com/georgexu99.png?size=96'
+slug: electron-21-0
+
+---
+
+Electron 21.0.0 has been released! It includes upgrades to Chromium `106`, V8 `10.4`, and Node.js `16.16.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 21.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### New Features
+
+TODO: Add features here
+
+### Stack Changes
+
+* Chromium `106`
+    * [New in Chrome 106](https://developer.chrome.com/blog/new-in-chrome-106/)
+    * [New in Chrome 105](https://developer.chrome.com/blog/new-in-chrome-105/)
+    * [New in DevTools](https://developer.chrome.com/blog/new-in-devtools-106/)
+* Node.js `16.16.0`
+    * [Node 16.16.0 blog post](https://nodejs.org/en/blog/release/v16.16.0/)
+* V8 `10.4`
+
+## Breaking & API Changes
+
+Below are breaking changes introduced in Electron 21. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/latest/breaking-changes) page.
+
+
+## End of Support for 18.x.y
+
+Electron 18.x.y has reached end-of-support as per the project's [support policy](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+| E18 (Mar'22) | E19 (May'22) | E20 (Aug'22) | E21 (Sep'22) | E22 (Dec'22) |
+| ------------ | ------------ | ------------ | ------------ | ------------ |
+| 18.x.y       | 19.x.y       | 20.x.y       | 21.x.y       | 22.x.y       |
+| 17.x.y       | 18.x.y       | 19.x.y       | 20.x.y       | 21.x.y       |
+| 16.x.y       | 17.x.y       | 18.x.y       | 19.x.y       | 20.x.y       |
+
+## What's Next
+
+In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8. Although we are careful not to make promises about release dates, our plan is to release new major versions of Electron with new versions of those components approximately every 2 months.
```

Comment:
IIRC, in recent releases this section was to discuss support changes -- e.g. when we jumped from 3 supported versions to 4 and also back to 3 again -- but since that has all played out, this section doesn't seem to be saying anything. Do we still need it?

If we went straight from the table above to the timeline link below, IMO that would still work

Reply:
Yeah, I think that's fair! I'll remove this paragraph